content/news/029/index.md

```diff
@@ -69,6 +69,26 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Engine Updates
 
+### [Rusty Engine 3.0]
+
+![An example Rusty Engine game](rusty_engine3.png)
+_The "Road Race" game prototype running under Rusty Engine 3.0_
+
+[Rusty Engine] by [Nathan Stocks] is a game engine for people who are learning Rust.
+
+Version 3.0 is a large release with many breaking changes. Notable new features include: [a full tutorial], custom asset loading (sprites, sounds, fonts), customizable game state, an interactive collider creator, and much more. See [the changelog for 3.0] for the full details.
```

Comment:
```suggestion
Version 3.0 is a large release with many breaking changes. Notable new features
include: [a full tutorial], custom asset loading (sprites, sounds, fonts), 
customizable game state, an interactive collider creator, and much more.
See [the changelog for 3.0] for the full details.
```

Reply:
Ah, I have my editor set to soft-wrap markdown. Forgot about explicitly adding the breaks. 🙈